__tests__/articles.test.js

```diff
@@ -0,0 +1,50 @@
+const db = require("../db/connection");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const {
+  topicData,
+  userData,
+  articleData,
+  commentData,
+} = require("../db/data/test-data");
+const request = require("supertest");
+const articlesData = require("../db/data/test-data/articles");
+
+beforeEach(() => {
+  return seed({ topicData, userData, articleData, commentData });
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("GET /api/articles/:article_id", () => {
+  it("200: should return an article object which contains articles by the given ':article_id' parameter", () => {
+    const test_id = 1;
+    return request(app)
+      .get(`/api/articles/${test_id}`)
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article[0].article_id).toEqual(test_id);
```

Comment:
the article we are getting back should be an object not an array with 1 object  Please edit so that the test looks for the data being sent back in this format